src/Tools/InstallUpdate/Program.cs

```diff
@@ -100,4 +168,19 @@ private static void RequestManualReinstall()
             Console.ReadKey();
         }
     }
+
+    public class WindowWrapper : System.Windows.Forms.IWin32Window
+    {
+        public WindowWrapper(IntPtr handle)
+        {
+            _hwnd = handle;
```

Comment:
We don't use underscore in our code to identify private variables.  Use `this.bla` in order to distinguish between `static private` fields and simply `private` fields.